src/PVRIptvData.cpp

```diff
@@ -87,17 +93,19 @@ PVR_ERROR PVRIptvData::GetCapabilities(kodi::addon::PVRCapabilities& capabilitie
   capabilities.SetSupportsTV(true);
   capabilities.SetSupportsRadio(true);
   capabilities.SetSupportsChannelGroups(true);
+  capabilities.SetSupportsProviders(true);
   capabilities.SetSupportsRecordings(false);
   capabilities.SetSupportsRecordingsRename(false);
   capabilities.SetSupportsRecordingsLifetimeChange(false);
   capabilities.SetSupportsDescrambleInfo(false);
+  capabilities.SetSupportsRecordings(true);
```

Comment:
On line 97 already the `capabilities.SetSupportsRecordings(false);` present, better there change to true.

Reply:
also seems then here a `  capabilities.SetSupportsRecordingsDelete(false);` missing.